Dockerfile.fuzz

```diff
@@ -0,0 +1,22 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0-alpine as builder
+
+RUN apk add --no-cache python3 py3-pip bash coreutils
+
+RUN wget -q "https://github.com/microsoft/restler-fuzzer/archive/refs/tags/v9.2.2.tar.gz"
+RUN tar -xvf v9.2.2.tar.gz && mv restler-fuzzer-9.2.2 restler-fuzzer
```

Comment:
combine these two runs and delete gz file after mv

Reply:
Actually, it is more efficient to just pipe the wget output into tar directly, and rename the resulting directory.

`wget -q -O - (url.tar.gz) | tar xz && mv`

Reply:
that's great